integration/networktest/scenarios/restart_validator_enclave.go

```diff
@@ -0,0 +1,75 @@
+package scenarios
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/obscuronet/go-obscuro/integration/networktest"
+
+	"github.com/obscuronet/go-obscuro/integration/networktest/functional"
+	"github.com/obscuronet/go-obscuro/integration/networktest/traffic"
+)
+
+type restartValidatorEnclaveScenarioTest struct{}
+
+func (r *restartValidatorEnclaveScenarioTest) Run(t *testing.T, network networktest.DevNetwork) error {
```

Comment:
for some reason this doesn't implement the `NetworkTest` interface.

since all seems to be compiling fine, it could mean that something is not 100% right

Reply:
Yeah the ScenarioTest thing is probably too confusing and unhelpful. This is a scenario test but not a `NetworkTest`, because the NetworkConnector interface is just for transacting against it doesn't allow you control the nodes.

I.e. ScenarioTest is a NetworkTest  but NetworkTest is **not** a ScenarioTest. Maybe ScenarioTest should have been called NodeOrchestrationTest or something.